staging/src/k8s.io/component-base/featuregate/feature_gate.go

```diff
@@ -268,7 +268,6 @@ func NewVersionedFeatureGate(emulationVersion *version.Version) *featureGate {
 	f.enabledRaw.Store(map[string]bool{})
 	f.emulationVersion.Store(emulationVersion)
 	f.queriedFeatures.Store(sets.Set[Feature]{})
-	klog.V(1).Infof("new feature gate with emulationVersion=%s", f.emulationVersion.Load().String())
```

Comment:
/lgtm

It's worth adding that this message is not actually shown in Kubernetes components like kube-controller-manager because logging is not initialized yet and thus at level 0 when this code is hit.

In general, code which gets invoked during the init phase should not log anything.